.github/workflows/update_iers.yml

```diff
@@ -0,0 +1,38 @@
+name: Auto-update IERS tables
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
```

Comment:
Did you change this from weekly to monthly?

Reply:
Yes, because I can now use the worklow dispatch for debugging if needed, so I can just set the cron to what we really want.